data_utils/connect.py

```diff
@@ -0,0 +1,49 @@
+from sqlalchemy import MetaData, Engine
+from sqlalchemy.ext.automap import automap_base, AutomapBase
+
+
+def get_automap_base_with_views(engine: Engine, schema: str) -> AutomapBase:
+    '''
+    Returns an AutomapBase object that also provides access to views of a schema
+
+    Parameters
+    ----------
+    engine : sqlalchemy.Engine
+        Engine that is already connected to a database
+    schema : str
+        Name of schema in the database for which we want to get an AutomapBase object
+
+    Returns
+    -------
+    AutomapBase
+        A declarative automap base for the schema of the database
+
+    Examples
+    --------
+    >>> from sqlalchemy import create_engine
+    >>> engine = create_engine('postgresql://user:password@localhost:5432/dbname'))
+    >>> Base = get_automap_base_with_views(engine=engine, schema='orca')
+    >>> print(type(Base))
+    <class 'sqlalchemy.ext.automap.AutomapBase'>
```

Comment:
This is great. You could also add one that uses the new function to get the engine.